source/firmware/xilinx-tools/flashing.md

```diff
@@ -74,9 +88,9 @@ Now the boot image has been loaded onto the PicoZed flash device. We need to con
 AMDC should now be programed and ready to go! Time to test.
 
 1.  Power cycle the board.
-2.  Ensure that `FPGA DONE` yellow LED comes on after ~1 second of powering up
+2.  Ensure that `FPGA DONE` yellow LED comes on after ~1 second of powering up, or the blink app begins and you see flashing LEDs
```

Comment:
Move this comment to another admonition below step 3. Something like:

```{caution}
If you see the `FPGA DONE` LED turn on, but your program appears to not be running, check the order of the boot partition with the above steps.
```

Reply:
done